src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -43,7 +43,7 @@
             .withEmail("anna@example.com").withAddress("4th street").build();
     public static final Person HENDRI = new PersonBuilder().withName("Hendri Tanjaya").withPhone("85108989")
             .withEmail("hendri@example.com").withAddress("PGPR")
-            .withEvent("Tutorial CS2103T", "2022-09-03", "10:00", "1").build();
+            .withEvent("Tutorial CS2103T", "2022-09-03", "10:00", "1", "W").build();
```

Comment:
Perhaps we could add a person with a biweekly event as well?

Reply:
I will probably add all the necessary stuff when I start adding the test cases, which I plan to do in a separate PR. Thanks for the suggestion!